### ISSUE TITLE
[Bug]: NPM & Setup 2FA

### ISSUE THREAD

**Issue by CodeWithDennis** [vormkracht10/filament-2fa#40]
### What happened?

While creating a reproduction repo to check for issue https://github.com/vormkracht10/filament-2fa/issues/36, I ran into various issues that my colleague also encountered. I made the repo to replicate the problem.

**1.** Running `npm run build` won't work but running `npm run dev` seems to work.

```
// Login Page
Unable to locate file in Vite manifest: vendor/vormkracht10/filament-2fa/resources/dist/filament-two-factor-auth.css.
```

**2.** Can't setup any 2FA

```
Method Illuminate\Foundation\Application::via does not exist.
```

### How to reproduce the bug

https://github.com/CodeWithDennis/filament-2fa-reproduction-issues

### Package Version

1.5

### PHP Version

8.2

### Laravel Version

11

### Which operating systems does with happen with?

macOS

**Comment by Baspa** [comment]
I couldn't reproduce this error using your repo:

`Unable to locate file in Vite manifest: vendor/vormkracht10/filament-2fa/resources/dist/filament-two-factor-auth.css.`

I rebuild the package to be sure, could you check if that's still happening? 

Steps I followed when using your reproduction repo:

1. `composer install && npm install`
2. `cp .env.example .env && php artisan key:generate`
3. Changes this code in the `User` model:

```php
public function canAccessPanel(Panel $panel): bool
{
    return true;
    // return str_ends_with($this->email, '@example.com');
}
```

The error: `Method Illuminate\Foundation\Application::via does not exist.` is fixed now in v1.5.1. @CodeWithDennis

**Comment by CodeWithDennis** [comment]
Its.. weird @Baspa 

https://github.com/user-attachments/assets/b3be90ec-f2d2-45b3-a5f6-cb79e0f4f5ff

**Comment by Baspa** [comment]
Aha, think I didn't read your comments right. You first run `npm run build` and then you get that error 🤔 

Gonna take a look at it right now @CodeWithDennis

**Comment by Baspa** [comment]
> Its.. weird @Baspa
> 
>  Schermopname.2024-09-19.om.20.27.00.mov

I'm 100% sure this is the cause:

`->viteTheme('vendor/vormkracht10/filament-2fa/resources/dist/filament-two-factor-auth.css');`

but haven't been able to find a solution yet.. I'm also not very sure if I need this actually.. @CodeWithDennis

**Comment by CodeWithDennis** [comment]
Unless you have custom styles you don't need it and can just add some instructions to add the paths to the tailwind config.

**Comment by CodeWithDennis** [comment]
Something like:

> Make sure you add the following to your `tailwind.config.js file.
> 
> './vendor/vormkracht10/filament-2fa/resources/**/*.blade.php',

**Comment by Baspa** [comment]
Should be fixed after I merge #42 @CodeWithDennis
